### PR TITLE
Make name and MITRE props consistent (SYN-6972)

### PR DESCRIPTION
**Pull request by mikemoritz**

### CODE REVIEW

File: synapse/models/risk.py [hunk 1]
```diff
@@ -313,6 +313,9 @@ def getModelDefs(self):
                     ('reporter:name', ('ou:name', {}), {
                         'doc': 'The name of the organization reporting on the mitigation.'}),
 
+                    ('mitre:attack:mitigation', ('it:mitre:attack:mitigation', {}), {
+                        'doc': 'A mapping to a Mitre ATT&CK mitigation if applicable.'}),
```
Comment (Cisphyx) [inline synapse/models/risk.py:317]: Should this be MITRE?

Comment (mikemoritz) [reply synapse/models/risk.py:317]: I went with how it was written on ou:campaign and ou:technique, but it's worth a go-back to see if we should change all of them.

**Review by invisig0th — DISMISSED** [review]

File: synapse/models/risk.py (hunk 1 above)
Comment (invisig0th) [reply synapse/models/risk.py:317]: I'm submitting a story to review the other docs and make them consistent

File: synapse/models/orgs.py [hunk 2]
```diff
@@ -542,9 +544,12 @@ def getModelDefs(self):
                     ('personnel:jobtitle', ('ou:jobtitle', {}), {
                         'doc': 'Title or designation for the contributed personnel.'}),
                 )),
+                ('ou:techname', {}, ()),
```
Comment (invisig0th) [inline synapse/models/orgs.py:547]: Lets leave this off for now. I'd like to possibly create a unified entity/idea name form :)

File: synapse/models/risk.py [hunk 3]
```diff
@@ -81,6 +81,8 @@ def getModelDefs(self):
                 ('risk:mitigation', ('guid', {}), {
                     'doc': 'A mitigation for a specific risk:vuln.',
                 }),
+                ('risk:mitname', ('str', {'lower': True, 'onespace': True}), {
```
Comment (invisig0th) [inline synapse/models/risk.py:84]: this one too :)  We can make an easy eventual migration that makes a form out of `:name` and `:names` :+1:

File: synapse/models/infotech.py [hunk 4]
```diff
@@ -381,39 +381,39 @@ def getModelDefs(self):
                     'doc': "A vulnerability scan result for an asset."}),
 
                 ('it:mitre:attack:status', ('str', {'enums': 'current,deprecated,withdrawn'}), {
-                    'doc': 'A Mitre ATT&CK element status.',
+                    'doc': 'A MITRE ATT&CK element status.',
```
Comment (vEpiphyte) [inline synapse/models/infotech.py:384]: I thought the preferred form was `MITRE` ?

Comment (mikemoritz) [reply synapse/models/infotech.py:384]: That's the change (Mitre->MITRE)...?

**Review by Cisphyx — APPROVED** [review]

File: synapse/models/infotech.py (hunk 4 above)
Comment (vEpiphyte) [reply synapse/models/infotech.py:384]: nothing to see here..

**Review by invisig0th — APPROVED** [review]

**Pull request merged**